src-tauri/src/main.rs

```diff
@@ -117,6 +121,14 @@ fn is_debug_mode() -> bool {
     return cfg!(debug_assertions);
 }
 
+#[allow(dead_code)]
+#[cfg(target_os = "linux")]
```

Comment:
Pretty sure this causes the compile issue on Windows. Would remove this and same with `linux_checks_librs()` and then just have `linux_checks_librs()` return an error or sth on Windows.

Reply:
Will try to get to later tonight, was really busy today, but eill do tommorow for sure 👍